### DISCUSSION TITLE
Blog post: Using HTML landmark roles to improve accessibility

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#383] [Content]
All feedback and comments related to https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/

**Comment by aardrian** [comment]
Thanks for making this discussion thinger.

I read the post and saw it was very Apple/Safari heavy by screen shots, linked resources, and the demonstration video. I asked about a PR on the socials because I wanted to add other screen reader user guides to the _Useful resources_ section, so I will do it here (but far more sloppily):

* [NVDA User Guide](https://www.nvaccess.org/files/nvda/documentation/userGuide.html) (the [_Browse Mode_](https://www.nvaccess.org/files/nvda/documentation/userGuide.html?#BrowseMode) section may be most useful for readers)
* [Getting Started with JAWS](https://www.freedomscientific.com/training/jaws/getting-started/) (the [_Web Browsing Keystrokes_](https://www.freedomscientific.com/training/jaws/hotkeys/#wb) also likely best for readers).
* [Narrator keyboard commands and touch gestures](https://support.microsoft.com/en-us/windows/appendix-b-narrator-keyboard-commands-and-touch-gestures-8bdab3f4-b3e9-4554-7f28-8b15bd37410a#WindowsVersion=Windows_11)
* [Use TalkBack gestures](https://support.google.com/accessibility/android/answer/6151827) and [Use TalkBack to browse the web with Chrome](https://support.google.com/accessibility/android/answer/2633135)
* [Orca Reading Commands](https://help.gnome.org/users/orca/stable/commands_reading.html.en) (though [_Structural Navigation Commands_](https://help.gnome.org/users/orca/stable/commands_structural_navigation.html.en) is probably more relevant for your post)

NVDA and JAWS are by far the most popular desktop screen readers, and generally we should encourage developers to test on those first as a result. I regularly counsel my clients to be wary of VoiceOver-only training materials for the same reason (never mind constant Safari bugs and VO quirks).

I yanked these from the screen readers portion of my post at https://adrianroselli.com/2022/11/your-accessibility-claims-are-wrong-unless.html#SRs.

Related, the videos offered by Tetralogical show screen readers in use in a non-technical way and they also offer a British Sign Language version; no idea if referencing those is kosher: https://tetralogical.com/blog/2021/09/29/browsing-with-a-desktop-screen-reader/

**Reply by schalkneethling** [reply]
Thanks so much, @aardrian. Adding these would be incredible.

**Reply by aardrian** [reply]
Ace. The passive voice suggests that someone other than you will be adding them. I don't have access, however.
